docs/howto/deployment/checklist.txt

```diff
@@ -36,6 +36,15 @@ Some of the checks described below can be automated using the :option:`check
 --deploy` option. Be sure to run it against your production settings file as
 described in the option's documentation.
 
+Switch away from ``manage.py runserver``
+=========================================
+
+    .. versionadded:: 5.2
```

Comment:
```suggestion
========================================
```
versionadded notes are for what is new in a given version, docs updates around advice that is still valid regardless of the version do not need these annotations